### PR TITLE
Fixes #207 - Added contactready dashboard

### LINKED ISSUE #207: Add contactready dashboard
Client side issue to match up with [server-filed issue](https://github.com/webcompat/webcompat-metrics-server/issues/126). Ooops!

### PR DESCRIPTION
**Pull request by laghee**
I still get weird linting errors every time (although I think these are different than before), but this works locally. (Note to self, when doing docs remember to mention updating static router as well as navigation container.)

Screenshot 2019-10-29 18 33 26

@karlcow Not a very exciting graph yet, but soon!

### CODE REVIEW

**Comment by laghee** [comment]
OK, not just me, then ...

But I didn't touch those files, so maybe eslint updated something?

**Comment by magsout** [comment]
@laghee lat me check soon ;)

**Comment by laghee** [comment]
(If you want me to change the order of those two import statements in another PR, @magsout, just let me know.)

**Comment by magsout** [comment]
@laghee waiting for https://github.com/webcompat/webcompat-metrics-client/pull/194 to be merged. After, we need to rebase.

**Comment by laghee** [comment]
@magsout Rebased! I screwed up something locally, but I'm pretty sure this should be fine (tests are passing anyway... 🤷‍♀)

**Comment by magsout** [comment]
@laghee the chart Failed. My fault, I updated the logic for containers. I can update the container if you want. My fault, sorry.

File: src/containers/ContactReady/index.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+import MetricsTemplate from "../MetricsTemplate";
+import { ObjectNested, Router } from "../../libraries";
+import {
+  mostAndLeast,
+  normalize,
+  getTemporaryDefaultFilters,
+} from "../../modules/Chart";
+import { TEMP_MIN_DATE } from "../../constants/Charts";
+
+const handleData = data => {
```
Comment (magsout) [inline src/containers/ContactReady/index.js]:
```
const handleData = data => {
  const localData = ObjectNested.get(data, "timeline", {});
  return {
    globalStats: mostAndLeast(localData),
    chart: normalize(localData, "openIssues"),
  };
};
```
the only change.

Comment (laghee) [reply src/containers/ContactReady/index.js]: @magsout OK, let me know if this is correct... 😁

**Pull request merged**